tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -56,6 +59,86 @@ public void CreateWwwAuthenticateResponse(string resource, string authorizationU
             Assert.IsNull(authParams.Error);
         }
 
+        [TestMethod]
+        [DataRow("AuthScheme", "" )]
+        [DataRow("AuthScheme", "realm = someRealm")]
+        [DataRow("AuthScheme", "token68")]
+        [DataRow("AuthScheme", "realm = someRealm token68")]

```

Comment:
This also looks wrong. I'd expect RawParameters to have `{"realm":"someRealm"}` ?

<img width="303" alt="image" src="https://user-images.githubusercontent.com/12273384/204033601-cce18a56-811c-4c3e-91ba-03052daa8606.png">